cypress/e2e/models/migration/migration-waves/migration-wave.ts

```diff
@@ -277,4 +276,44 @@ export class MigrationWave {
         cy.contains("li", date.toLocaleString("en-us", { month: "long" })).click();
         cy.contains("button:not([disabled])", `${date.getDate()}`).first().click();
     }
+
+    public clickOnApplicationStatus() {
+        cy.contains(this.name)
+            .parents("tr")
+            .within(() => {
+                cy.get(MigrationWaveView.applicationStatusColumn).click();
+            });
+    }
+
+    public removeApplication(applicationName) {
+        cy.contains(this.name)
```

Comment:
Can we use https://github.com/konveyor/tackle-ui-tests/blob/main/cypress/utils/utils.ts#L563 

expandRowDetails() here .

Reply:
I don't think this will work, in my case I have to explicitly click on the status column for the wave